tests/Doctrine/Tests/DBAL/ConnectionTest.php

```diff
@@ -153,9 +158,8 @@ public function testConnectDispatchEvent()
         $driverMock = $this->createMock(Driver::class);
         $driverMock->expects($this->at(0))
                    ->method('connect');
-        $platform = new Mocks\MockPlatform();
 
-        $conn = new Connection(['platform' => $platform], $driverMock, new Configuration(), $eventManager);
+        $conn = new Connection([], $driverMock, new Configuration(), $eventManager);
```

Comment:
Was `platform` removed before it was unused or for some other reason?

Reply:
It wasn't used in the test.